### ISSUE TITLE
Not using 1.0.0

### ISSUE THREAD

**Issue by kasper9n** [oven-sh/setup-bun#20]
The action is downloading `0.5.6`. When I explicitly set `bun-version` to 1.0.0, it still downloads `0.5.6`.

Log output:
```
Run oven-sh/setup-bun@v1
  with:
    bun-version: 1.0.0
Downloading a new version of Bun: https://bun.sh/download/0.5.6/linux/x64?avx2=true&profile=false
/usr/bin/unzip -o -q /home/runner/work/_temp/aecc4a[3](https://github.com/probablykasper/kyper/actions/runs/6151443142/job/16691722213#step:3:3)7-5cc8-[4](https://github.com/probablykasper/kyper/actions/runs/6151443142/job/16691722213#step:3:4)99d-b88e-3be91f1e9efa
/home/runner/.bun/bin/bun --revision
bun: a fast bundler, transpiler, JavaScript Runtime and package manager for web software.

  run       ./my-script.ts        Run JavaScript with bun, a package.json script, or a bin
  x         bun-repl              Install and execute a package bin (bunx)

  init                            Start an empty Bun project from a blank template
  create    next ./app            Create a new project from a template (bun c)
  install                         Install dependencies for a package.json (bun i)
  add       @zarfjs/zarf          Add a dependency to package.json (bun a)
  link                            Link an npm package globally
  remove    redux                 Remove a dependency from package.json (bun rm)
  unlink                          Globally unlink an npm package
  pm                              More commands for managing packages

  dev       ./a.ts ./b.jsx        Start a bun (frontend) Dev Server
  bun       ./a.ts ./b.jsx        Bundle dependencies of input files into a .bun

  upgrade                         Get the latest version of bun
  completions                     Install shell completions for tab-completion
  discord                         Open bun's Discord server
  help                            Print this help menu
/home/runner/.bun/bin/bun --version
0.[5](https://github.com/probablykasper/kyper/actions/runs/6151443142/job/16691722213#step:3:5).[6](https://github.com/probablykasper/kyper/actions/runs/6151443142/job/16691722213#step:3:6)
/usr/bin/tar --posix -cf cache.tzst --exclude cache.tzst -P -C /home/runner/work/kyper/kyper --files-from manifest.txt --use-compress-program zstdmt
Cache Size: ~29 MB (303[7](https://github.com/probablykasper/kyper/actions/runs/6151443142/job/16691722213#step:3:7)[9](https://github.com/probablykasper/kyper/actions/runs/6151443142/job/16691722213#step:3:9)[34](https://github.com/probablykasper/kyper/actions/runs/6151443142/job/16691722213#step:3:35)7 B)
Cache saved successfully
```

**Comment by Electroid** [comment]
Hi there, this was fixed in #19. Sorry for the confusion!

**Comment by kasper9n** [comment]
Ah sorry for the duplicate issue! That was quick
